apps/costcontrol/js/costcontrol.js

```diff
@@ -618,12 +618,14 @@ var CostControl = (function() {
       result.status = 'success';
       result.data = lastDataUsage;
 
-      // Once bug 1083680 is solved, both caches should contain the same values
-      // so we could use only `costcontrol.lastDataResults`.
-      if (perApp) {
-        costcontrol.lastDataResultsPerApp = lastDataUsage;
-      } else {
-        costcontrol.lastDataResults = lastDataUsage;
+      if (costcontrol) {
```

Comment:
Is this additional guard related with another bug. If so, file a new one and fix it there.